policies-procedures-and-docs/communications/tender-template.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: page
+title: Tender template
+image: https://images.unsplash.com/photo-1683178861337-ca70ef8c0db3
+photos:
+  name: Zetong Li
+  license: CC-BY
+  url: https://images.unsplash.com/photo-1683178861337-ca70ef8c0db3
+---
+
+Looking for someone to do short-term consultancy work? Here's a template to kickstart your announcement. 
```

Comment:
```suggestion
Looking for someone to do **short-term consultancy** work? Here's a template to kickstart your announcement. 
```
```suggestion
Looking for someone to do **short-term consultancy** work? Here's a **template** to kickstart your announcement. 
```

Reply:
Makes the keywords stand out, conforming to our existing standards. (_see PR template_)